src/app/core/internal/view/WireAssembler.ts

```diff
@@ -0,0 +1,50 @@
+import {BezierCurve} from "math/BezierCurve";
+
+import {Schema} from "core/schema";
+
+import {Assembler}       from "./Assembler";
+import {BezierCurvePrim} from "./rendering/prims/BezierCurvePrim";
+
+
+export class WireAssembler extends Assembler<Schema.Wire> {
+    protected assembleCurveStyle(wire: Schema.Wire) {
+        return this.options.wireStyle(
+            this.selections.has(wire.id),
+            (wire.props.color ?? "#ffffff"),
```

Comment:
Should this somehow be assigned or connected with the `DefaultRenderOptions` below?

Reply:
Yeah, it uses `this.options` which is an instance of `DefaultRenderOptions`